src/components/Characters/Characters.test.jsx

```diff
@@ -0,0 +1,27 @@
+import { screen, render } from '@testing-library/react'
+import Characters from './Characters'
+import { Link } from 'react-router-dom'
+import { MemoryRouter } from 'react-router-dom'
+
+describe('characters component test', () => {
+    it('should render character component', () =>{
+
+        const mockCharacters = [
```

Comment:
this should be like: 
`[{id: 1, name: 'Summer Smith', image: 'www.example.com'},{id: 2,name: 'Alan Rails',image: 'www.example.com'}]`